### PR TITLE
CI: temporarily disable interval in feather tests (pyarrow 0.17.1 regression)

### PR DESCRIPTION
**Pull request by jorisvandenbossche**
xref https://github.com/pandas-dev/pandas/issues/34255

Temporary solution to get green CI.

### CODE REVIEW

File: pandas/tests/io/test_feather.py [hunk 1]
```diff
@@ -83,7 +83,9 @@ def test_basic(self):
         if pyarrow_version >= LooseVersion("0.16.1.dev"):
             df["periods"] = pd.period_range("2013", freq="M", periods=3)
             df["timedeltas"] = pd.timedelta_range("1 day", periods=3)
-            df["intervals"] = pd.interval_range(0, 3, 3)
+            # TODO temporary disable due to regression in pyarrow 0.17.1
```
Comment (jbrockmendel) [inline pandas/tests/io/test_feather.py:86]: Is there a condition when this can be restored?  Eg “When X version Y is released”

Comment (jorisvandenbossche) [reply pandas/tests/io/test_feather.py:86]: It depends on when it will be fixed in pyarrow, so difficult to say for sure at this moment

Comment (jbrockmendel) [reply pandas/tests/io/test_feather.py:86]: ok thanks.  LGTM pending green

**Comment by jbrockmendel** [comment]
Travis is green, just hasn't propagated the icon here

**Pull request merged**